pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
+- **Heartbeat Available, but cloudlink not**: The device is unable to connect to cloudlink (e.g. a firewall is blocking VPN traffic), and won't be able to provide remote SSH connections. A device with Heartbeat Only status has internet connectivity and can poll the cloud for new updates to apply. However, performing any actions which do not change the target state or attempting to access the device via SSH will error, as these actions are proxied via the cloudlink tunnel connection. Actions that do not change the target state include purging data, restarting, rebooting, or shutting down.
 
-__Note:__ If the device is powered off or loses all network connectivity, it will remain in the `Online (Heartbeat Only)` state until the last target state fetch exceeds the device [API polling interval][poll-interval].
+__Note:__ If the device is powered off or loses all network connectivity, the `heartbeat` indictor show as connected (green) until the last target state fetch exceeds the device [API polling interval][poll-interval], at which time it will be marked as `Disconnected`.
 
 ## Debugging Device Status
 
 If you find your device to be displaying a status which is unclear even with the above notes, visit the [device debugging masterclass][debugging-masterclass] for more information.
 
+# Update Status
+
+The `Update status` and `Update status duration` fields on devices represent the state of the current application update to target state. 
+
+__Note:__ Data for these states will only get populated for devices with supervisor 16.7.0 or later.
+
+## Update statuses
+
+| Status                                       | Description                                                                                                                                                                      |
+|----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Done**                              | The update, restart or configuration change is complete and the device is in its target state.                                          |
+| **Downloading**                       | The supervisor is downloading one or more container images associated with the target release                                           |
+| **Downloaded**                        | All container images in the target release have been downloaded. **Note:** If the device holds an update lock, it will stop in this state until the lock is released (depending on the [update strategy](https://docs.balena.io/learn/deploy/release-strategy/update-strategies/#controlling-the-update-strategy)). |
```

Comment:
Maybe put the note on a separate line here

Reply:
unfortunately it doesn't seem possible to do within markdown. I'm sure there is a way to do it with html in markdown but it seems non-trivial :/